sync3/handler/ensure_polling.go

```diff
@@ -129,7 +141,7 @@ func (p *EnsurePoller) OnInitialSyncComplete(payload *pubsub.V2InitialSyncComple
 		log.Trace().Msg("OnInitialSyncComplete: we weren't waiting for this")
```

Comment:
Should we promote this to a warning?

Reply:
Yes this spams on startup, so trace is fine.